tests/backends/test_results.py

```diff
@@ -43,25 +43,29 @@ def test_keccak_256(keccak, prehash, expected_result):
             [b'', b'', b''],
             b"\xc5\xd2F\x01\x86\xf7#<\x92~}\xb2\xdc\xc7\x03\xc0\xe5\x00\xb6S\xca\x82';\x7b\xfa\xd8\x04]\x85\xa4p",  # noqa: E501
         ),
-        (
+        pytest.param(
             [bytearray(b''), bytearray(b''), bytearray(b'')],
             b"\xc5\xd2F\x01\x86\xf7#<\x92~}\xb2\xdc\xc7\x03\xc0\xe5\x00\xb6S\xca\x82';\x7b\xfa\xd8\x04]\x85\xa4p",  # noqa: E501
+            marks=pytest.mark.xfail(reason="pycryptodome does not support updating w/ bytearrays"),
         ),
         (
             [b'arst', b'tsra'],
             b"\xb1\xf3T\xb2\x8f\xf2\x84R\xd6\xb9\xd6\x1fA\x06\x1b\xbe\x82\xbe\xb1\xfc\x98\xf33d\xa8\x05\x8d\x1a]\x16M\x05",  # noqa: E501
         ),
-        (
+        pytest.param(
             [bytearray(b'arst'), bytearray(b'tsra')],
             b"\xb1\xf3T\xb2\x8f\xf2\x84R\xd6\xb9\xd6\x1fA\x06\x1b\xbe\x82\xbe\xb1\xfc\x98\xf33d\xa8\x05\x8d\x1a]\x16M\x05",  # noqa: E501
+            marks=pytest.mark.xfail(reason="pycryptodome does not support updating w/ bytearrays"),
         ),
-        (
+        pytest.param(
             [bytearray(b'arst'), b'tsra'],
             b"\xb1\xf3T\xb2\x8f\xf2\x84R\xd6\xb9\xd6\x1fA\x06\x1b\xbe\x82\xbe\xb1\xfc\x98\xf33d\xa8\x05\x8d\x1a]\x16M\x05",  # noqa: E501
+            marks=pytest.mark.xfail(reason="pycryptodome does not support updating w/ bytearrays"),
         ),
-        (
+        pytest.param(
             [b'arst', bytearray(b'tsra')],
             b"\xb1\xf3T\xb2\x8f\xf2\x84R\xd6\xb9\xd6\x1fA\x06\x1b\xbe\x82\xbe\xb1\xfc\x98\xf33d\xa8\x05\x8d\x1a]\x16M\x05",  # noqa: E501
+            marks=pytest.mark.xfail(reason="pycryptodome does not support updating w/ bytearrays"),
```

Comment:
Can't we normalize at the `eth-hash` level?

Reply:
you mean stop accepting bytearrays for any backend?

Reply:
no, I mean wrap the input value in `bytes` to coerce `bytearray -> bytes` before it gets to pycryptodome.

Reply:
Actually, it's moot, because my problem was a local old install of pycryptodome. False alarm.